rialto_airflow/harvest/authors.py

```diff
@@ -75,7 +75,7 @@ def to_boolean(value: str) -> bool:
     bool_map = {"true": True, "false": False}
 
     # will raise KeyError if unexpected value
-    return bool_map.get(value.strip().lower())
+    return bool_map[value.strip().lower()]
```

Comment:
turns out `get` can return `None`, and the square brackets will give the `KeyError` if the thing is missing (also tested in the REPL)